source/_integrations/abode.markdown

```diff
@@ -13,6 +13,7 @@ ha_category:
   - Switch
 ha_release: 0.52
 ha_iot_class: Cloud Push
+ha_quality_scale: silver
```

Comment:
This is taken from the integration manifest. Open a backend PR instead for this change.